From 7b8a8a9ff39172621f2cc89c3a1701bc64da3cc3 Mon Sep 17 00:00:00 2001
From: Guochun Huang <hero.huang@rock-chips.com>
Date: Thu, 23 May 2024 10:58:45 +0800
Subject: [PATCH 298/305] ARM: rockchip_defconfig: enable
 CONFIG_VIDEO_ROCKCHIP_HDMIRX_CLASS for rk628

Type: Fix/Function
Redmine ID: #
Associated modifications: gerrit links
Test: test method

Signed-off-by: Guochun Huang <hero.huang@rock-chips.com>
Change-Id: I06d244be1f15036cc32ee6751af8e14397a020b0
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 8e52d7ea518c8c..e42e78630e0750 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -417,6 +417,7 @@ CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
 CONFIG_VIDEO_ROCKCHIP_CIF=y
+CONFIG_VIDEO_ROCKCHIP_HDMIRX_CLASS=y
 CONFIG_VIDEO_ROCKCHIP_ISP1=y
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
 CONFIG_VIDEO_GC2145=y
-- 
2.17.1

